visidata/loaders/archive.py

```diff
@@ -16,9 +16,19 @@ class ZipSheet(Sheet):
         ColumnAttr('compress_size', type=int)
     ]
 
+    def open_zipfile(self, fp, *args, **kwargs):
```

Comment:
We should use something other than `open_`, unless we want `-f zipfile` to work somehow (and then this would have to return a sheet of some kind).  Maybe `openZipFile`?